python/mach_bootstrap.py

```diff
@@ -195,6 +195,14 @@ def bootstrap(topdir):
         print('Current path:', topdir)
         sys.exit(1)
 
+    # We don't support MinGW Python
+    if os.path.join(os.sep, 'mingw64', 'bin') in sys.executable:
+        print('Cannot run mach with MinGW Python.')
+        print('\nPlease execute following commands:')
+        print(' mv /mingw64/bin/python2.exe /mingw64/bin/python2-mingw64.exe')
+        print(' mv /mingw64/bin/python2.7.exe /mingw64/bin/python2.7-mingw64.exe')
```

Comment:
Nit: We don't have to ask them to follow a specific command. The message could be something like,

```
Please rename /mingw64/bin/python2.exe to /mingw64/bin/python2-mingw64.exe and /mingw64/bin/python2.7.exe to /mingw64/bin/python2.7-mingw64.exe
```